docs/user-guide.md

```diff
@@ -1,8 +1,47 @@
 ## User Guide
 
 ### Installation
```

Comment:
Maybe we should move 

https://github.com/nix-community/nixd/blob/db1a463720d4e5d569b52b0c9af2e6c4736e8608/docs/user-guide.md?plain=1#L49

here to recommend users compile from source. Version 1.0.0 is very experimental and has some known bugs.